reviewkitteh/README.md

```diff
@@ -16,7 +16,7 @@ This project automates the process of listening for GitHub pull request events a
 ### Configure integrations
 
 > [!IMPORTANT]
->  The `autokitteh.yaml` file includes environment variables for the GitHub, Slack, and Google Sheets connections that need to be configured.
+>  The `autokitteh.yaml` file includes environment variables for the GitHub, Slack, and Google Sheets connections that need to be configured. When setting the `CHANNEL` variable, be sure to use the channel ID, not the channel name.
```

Comment:
1. The manifest has `CHANNEL_ID`, not `CHANNEL` like in the original project
2. But why just an ID? Are you sure a name wouldn't work?

Reply:
Yes, apparently `chat.update` only accepts ID's although this isn't well documented.

https://github.com/slack-ruby/slack-ruby-client/issues/270#issuecomment-492279873

All I know, is `chat.update` couldn't find the channel until I fed it an ID.